setup.py

```diff
@@ -85,12 +86,21 @@
     datafiles = [os.path.join('data',os.path.basename(f)) for f in datafiles]
     package_data = {'pyproj':datafiles}
 
+# store pyproj version information (from _proj.pyx) in version variable
+# (taken from Fiona)
+with open('_proj.pyx', 'r') as f:
+    for line in f:
+        if line.find("__version__") >= 0:
+            version = line.split("=")[1].strip()
+            version = version.strip('"')
+            version = version.strip("'")
```

Comment:
Instead of calling strip twice you can do `version = version.strip()[1:-1]`.  Also, I'd rather use `break` than continue, you do not need to keep looking the rest of the file :wink:

:+1: BTW